Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Isotropic_remeshing/remesh_impl.h

```diff
@@ -199,14 +200,15 @@ namespace internal {
                                       PMP::parameters::edge_is_constrained_map(ecmap)
                                      .face_index_map(fimap));
         if(nb_cc == 1){
-          CGAL::internal::remove_property(patch_ids_map, pmesh);
+          // CGAL::remove_property(patch_ids_map, pmesh);
```

Comment:
remove comment?

Reply:
I will remove it once I have made a code review with @janetournois

Reply:
done